### DISCUSSION TITLE
How to fix it when pmv returns nan?

### DISCUSSION THREAD

**Discussion by RYAH3YH** [CenterForTheBuiltEnvironment/pythermalcomfort#148] [General]
The range of pmv is only [-2,2], but in the process of building simulation or reinforcement learning control, sometimes there will be greater than 2 or less than -2, which is returned nan, how to solve this?

**Comment by FedericoTartarini** [comment]
Please have a look at the official documentation, you can get the results beyond [-2,2] by setting limit_inputs=False
